pkg/store/gcworker/gc_worker.go

```diff
@@ -394,9 +401,7 @@ func (w *GCWorker) leaderTick(ctx context.Context) error {
 	// Note that when `keyspace-name` is set, `checkLeader` will be done within the key space.
 	// Therefore only one TiDB node in each key space will be responsible to do delete range.
 	// TODO: Use result of AdvanceTxnSafePoint instead, which makes it unnecessary to manually check the config.
-	const cfgGCManagementType = "gc_management_type"
-	const cfgGCManagementTypeKeyspaceLevel = "keyspace_level"
-	if keyspaceMeta := w.store.GetCodec().GetKeyspaceMeta(); keyspaceMeta != nil && keyspaceMeta.Config[cfgGCManagementType] != cfgGCManagementTypeKeyspaceLevel {
+	if w.isUnifiedGC() {
```

Comment:
It seems that the comment needs to be updated.
But it's out of the topic of this PR. I'll do it later.